dmidecode ONIE cross compile patch

Copyright (C) 2013 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Small modifications to the dmidecode Makefile for cross compiling in
the ONIE build environment.

Main changes:

- prefix gcc  with $(CROSS_PREFIX)
Summary:

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..f03a162
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,3 @@
+*.o
+biosdecode
+dmidecode
diff --git a/Makefile b/Makefile
index 55378bb..3913a44 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 #   (at your option) any later version.
 #
 
-CC      = gcc
+CC      = $(CROSS_COMPILE)gcc
 CFLAGS  = -W -Wall -Wshadow -Wstrict-prototypes -Wpointer-arith -Wcast-qual \
           -Wcast-align -Wwrite-strings -Wmissing-prototypes -Winline -Wundef
 #CFLAGS += -DBIGENDIAN
@@ -25,6 +25,9 @@ CFLAGS += -O2
 # Pass linker flags here
 LDFLAGS =
 
+CFLAGS	+= $(ONIE_CXXFLAGS)
+LDFLAGS	+= $(ONIE_LDFLAGS)
+
 DESTDIR =
 prefix  = /usr/local
 sbindir = $(prefix)/sbin
